README.md

```diff
@@ -110,7 +110,17 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")? "moon goodnight"
+- What is mystery6("Ada Developers Academy")? "Academy Developers Ada"
+- What is mystery6("Hi, there!")? "there! Hi,"
```

Comment:
Minor, but notable: All of these answers should have a leading space at the beginning " moon goodnight", " Academy Developers Ada", " there! Hi,"